config-model/src/main/java/com/yahoo/vespa/model/admin/clustercontroller/ClusterControllerContainer.java

```diff
@@ -110,7 +110,9 @@ public void getConfig(ZookeeperServerConfig.Builder builder) {
 
     @Override
     public void getConfig(QrStartConfig.Builder builder) {
-        builder.jvm(new QrStartConfig.Jvm.Builder().heapsize(512));
+        builder.jvm.heapsize(512);
+        builder.jvm.availableProcessors(2);
+        builder.jvm.verbosegc(false);
```

Comment:
Set heapSizeAsPercentageOfPhysicalMemory to 0 here as done for metrics proxy as well?

Reply:
Correct, but that means that a scenario is not tested. I will add a test for hosted too.